packages/es-components/src/index.js

```diff
@@ -22,6 +22,8 @@ export {
   DrawerItemContext
 } from './components/containers/drawer/Drawer';
 
+export useUniqueId from './components/util/useUniqueId';
```

Comment:
Could you drop this down with the rest of the `./util` exports? This way we have consistent groupings